app/src/main/java/fr/gaulupeau/apps/Poche/network/ImageCacheUtils.java

```diff
@@ -38,9 +38,9 @@
             = Pattern.compile("<img[\\w\\W]*?src=\"([^\"]+?)\"[\\w\\W]*?>");
 
     private static final Pattern[] responsiveParametersPatterns = {
-            Pattern.compile("srcset=\".*\""),
-            Pattern.compile("sizes=\".*\""),
-            Pattern.compile("data-zoom-src=\".*\""),
+            Pattern.compile("srcset=\"[^\"]*?\""),
```

Comment:
Good catch. my regex was the greedy one which is wrong in this case. I used the wrong quantifier. `[^\"]*?` looks complicated on the first view until i understood, that this means all chars except double quotes. `.*?` should work for the whole pattern, too, shouldn't it? But i think your expression is more explicit telling to not catch double quotes.